src/App.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
-import './App.css';
+import { useSelector } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
+import SplashScreen from './components/SplashScreen/SplashScreen';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```

Comment:
- [ ] When I go to the "Delete doctors" section, the "Doctors" and "Add Doctor" links dissapear.  The test is still there but its not clickable anymore:

![image](https://user-images.githubusercontent.com/57421823/206867129-6ad3c09b-0274-47f0-88b5-28e17a822a9e.png)

Kindly correct this bug.